### PR TITLE
Instant retrieval tier

### PR DESCRIPTION
**Pull request by efirs**

### CODE REVIEW

File: docs/objects/tiers.md [hunk 1]
```diff
@@ -43,6 +48,7 @@ Tigris accepts S3 compatible storage classes:
 - STANDARD: for Standard tier
 - STANDARD_IA: for Infrequent Access tier
 - GLACIER: for Archive tier
+- GLACIER_IR: for Archive instant retrieval tier
```
Comment (ovaistariq) [inline docs/objects/tiers.md:51]: Is this the same class that S3 uses?

File: docs/pricing/index.md [hunk 2]
```diff
@@ -11,15 +11,15 @@ Tigris pricing is based on the following components:
...
-| Data Storage                                  | $0.02/GB/month              | $0.01/GB/month              | $0.004/GB/month             |
-| Class A Requests: PUT, COPY, POST, LIST       | $0.005/1000 requests        | $0.005/1000 requests        | $0.005/1000 requests        |
-| Class B Requests: GET, SELECT, and all others | $0.0005/1000 requests       | $0.0005/1000 requests       | $0.0005/1000 requests       |
-| Data Retrieval                                | Free                        | $0.01/GB                    | Free                        |
-| Minimum Storage Retention                     | -                           | 30 days                     | 90 days                     |
-| Object Notifications                          | $0.01/1000 events published | $0.01/1000 events published | $0.01/1000 events published |
-| Egress (Data Transfer to Internet)            | Free                        | Free                        | Free                        |
+| Component                                     | Standard Tier               | Infrequent Access Tier      | Archive Tier \*\*           | Archive<br/>instant retrieval<br/>tier |
```
Comment (ovaistariq) [inline docs/pricing/index.md:14]: Don't use newlines `<br/>` that would mess with the table layout.

File: docs/objects/tiers.md (hunk 1 above)
Comment (efirs) [reply docs/objects/tiers.md:51]: Yes

File: docs/objects/tiers.md [hunk 3]
```diff
@@ -23,6 +23,11 @@ Low-cost storage for data archiving with infrequent access. The data is not
 immediately available for access and requires restoration before it can be
 accessed. Restoration time is typically around 1 hour.
 
+## Archive with instant retrieval tier
+
+Low-cost storage for data archiving with infrequent access, offering rapid
+access when needed. The data is immediately available for access.
```
Comment (ovaistariq) [inline docs/objects/tiers.md:29]: The data is immediately available for access and does not need to go through a restoration step.

**Review by ovaistariq — APPROVED** [review]

**Pull request merged**